File: sandbox/python/src/ApiAppGetDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.ApiAppApi(api_client).api_app_get(
            client_id="0dd3b823a682527788c4e40cb7b6f7e9",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ApiApp#api_app_get: %s\n" % e)

File: sandbox/node/src/ApiAppGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.ApiAppApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

const clientId = "0dd3b823a682527788c4e40cb7b6f7e9";

apiCaller.apiAppGet(
    clientId,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ApiApp#apiAppGet:");
  console.log(error.body);
});
